Bottom: faa29a50e2621233c8991345e63c19943601f04e
Top:    e7e4e262ec89f12cd4084b3e554e43ad81eb2974
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-27 19:25:30 +0000

Refresh of find-c-avoid-modifying-bounds-

---

diff --git a/src/find.c b/src/find.c
index 314abb9..6ba3009 100644
--- a/src/find.c
+++ b/src/find.c
@@ -1749,14 +1749,14 @@ LookupLOConnectionsToArc (ArcType *Arc, Cardinal LayerGroup)
           info.layer = layer;
           /* add arcs */
           if (setjmp (info.env) == 0)
-            r_search (LAYER_PTR (layer)->line_tree, &search_box, NULL,
-                      LOCtoArcLine_callback, &info);
+            r_search (LAYER_PTR (layer)->line_tree, &search_box,
+                      NULL, LOCtoArcLine_callback, &info);
           else
             return true;
 
           if (setjmp (info.env) == 0)
-            r_search (LAYER_PTR (layer)->arc_tree, &search_box, NULL,
-                      LOCtoArcArc_callback, &info);
+            r_search (LAYER_PTR (layer)->arc_tree, &search_box,
+                      NULL, LOCtoArcArc_callback, &info);
           else
             return true;
 
@@ -1889,14 +1889,14 @@ LookupLOConnectionsToLine (LineType *Line, Cardinal LayerGroup,
           info.layer = layer;
           /* add lines */
           if (setjmp (info.env) == 0)
-            r_search (LAYER_PTR (layer)->line_tree, &search_box, NULL,
-                      LOCtoLineLine_callback, &info);
+            r_search (LAYER_PTR (layer)->line_tree, &search_box,
+                      NULL, LOCtoLineLine_callback, &info);
           else
             return true;
           /* add arcs */
           if (setjmp (info.env) == 0)
-            r_search (LAYER_PTR (layer)->arc_tree, &search_box, NULL,
-                      LOCtoLineArc_callback, &info);
+            r_search (LAYER_PTR (layer)->arc_tree, &search_box,
+                      NULL, LOCtoLineArc_callback, &info);
           else
             return true;
           /* now check all polygons */
